tests/scenarios/vite-internals-test.ts

```diff
@@ -321,7 +321,8 @@ function runViteInternalsTest(scenario: Scenario) {
     });
 
     Qmodule('vite optimize', function () {
-      test('vite optimize should succeed', async function (assert) {
+      // skipped due to https://github.com/vitejs/vite/issues/19316
+      QUnit.skip('vite optimize should succeed', async function (assert) {
```

Comment:
This is the only unpleasant surprise so far.  https://github.com/vitejs/vite/issues/19316

Reply:
This has now been fixed upstream and will be released in vite@6.1.0 soon, I have updated our CI to target the corresponding beta

Reply:
correction: the fix didn't get all the way to where we needed to 😞 Keeping an eye on upstream